tests.cfg

```diff
@@ -91,6 +91,8 @@ test-eggs =
     plone.theme
     plone.transformchain
     plone.transformchain [test]
+    plone.volto
+    plone.volto [test]
```

Comment:
Is it necessary to add both `plone.volto` and `plone.volto [test]`?
I think `plone.volto [test]` is enough